packages/cli/src/services/util.ts

```diff
@@ -37,8 +41,14 @@ export async function walkDirectory (
 
 export async function loadJsFile (filepath: string): Promise<any> {
   try {
-    // eslint-disable-next-line @typescript-eslint/no-var-requires
-    let exported = require(filepath)
+    // There is a Node opened issue related with a segmentation fault using ES6 modules
+    // with jest https://github.com/nodejs/node/issues/35889
+    // As a work around, we check if Jest is running to modify the way to import the module.
+    // TODO: investigate if the issue is fixed to clean up the conditional import
+    let { default: exported } = process.env.JEST_WORKER_ID !== undefined
```

Comment:
Instead of using the process env, should we try this `typeof jest !== 'undefined'`? This is in case anyone sets the jest worked id as an env 🤷 

I got the idea from here https://stackoverflow.com/questions/50940640/how-to-determine-if-jest-is-running-the-code-or-not

Reply:
Done. Good point. I also got the idea from the same post. I like the way you suggest, it seems more secure.